src/pika_rm.cc

```diff
@@ -512,6 +515,39 @@ uint32_t SyncMasterPartition::ConsensusTerm() {
 
 void SyncMasterPartition::ConsensusUpdateTerm(uint32_t term) {
   coordinator_.UpdateTerm(term);
+  CommitPreviousLogs(term);
+}
+
+void SyncMasterPartition::CommitPreviousLogs(const uint32_t& term) {
+  if (HasUncommittedLogs(term)) {
+    AppendDummyLog();
+  }
+}
+
+bool SyncMasterPartition::HasUncommittedLogs(const uint32_t& current_term) {
```

Comment:
I think we don't need this logic for now, cause leader just send dummy whenever it increase its term.